tests/e2e/specs/BoundaryTimerEvent.spec.js

```diff
@@ -18,8 +14,8 @@ import { nodeTypes } from '../support/constants';
 import { defaultNodeColor, invalidNodeColor } from '../../../src/components/nodeColors';
 
 const boundaryEventSelector = '.main-paper ' +
-    '[data-type="processmaker.components.nodes.task.Shape"] + ' +
-    '[data-type="processmaker.components.nodes.boundaryEvent.Shape"]';
+  '[data-type="processmaker.components.nodes.task.Shape"] + ' +
+  '[data-type="processmaker.components.nodes.boundaryEvent.Shape"]';
 
 describe('Boundary Timer Event', () => {
   beforeEach(() => {
```

Comment:
I think the _Render a boundary timer event_ test below can be added to the group of common tests as well, since the `boundaryTimerEventXML` it tests is the same as the one you provided for `eventXMLSnippet` in `BoundaryEventCommonBehaviour.spec.js`.

Reply:
@chipit24 this test has now been moved too.